funds/privacy_guides/privacy_guides_funds.md

```diff
@@ -0,0 +1,163 @@
+* Any decisions that would violate the restrictions in **Appendix A**.
+
+## MAGIC Fund Committee Ethical and Legal Violations
+
+If the MAGIC Grants board determines (in its sole discretion) that any MAGIC Fund committee member has materially violated any MAGIC Grants policies, or is a compliance or legal risk to MAGIC Grants, MAGIC Grants may immediately remove the relevant committee member.
+
+## MAGIC Fund Committee Member Impeachment
+
+*Any details in **Appendix C** supersede the relevant paragraphs in this section.*
+
+If MAGIC Fund Voters wish to impeach an elected committee member, they can call for a special vote. Three quarters of all MAGIC Fund Voters must vote for the committee member’s impeachment (for clarity: not three quarters of just the voter turnout). Should this threshold be met, the MAGIC Grants Board shall evaluate the situation and determine in its own discretion whether to remove the committee member or not.
+
+Committee members can vote to impeach another committee member. All other committee members must vote for the committee member's impeachment. Should this occur, the MAGIC Grants board shall evaluate the situation and determine in its own discretion whether to remove the committee member or not.
+
+## Fund Disbursement Requirements
+
+MAGIC Grants will assist with the proper disbursement of funds. In general, funds must be paid after completion of relevant contract work, or with controls as required by law. Contractors will be expected to sign a contract that covers the scope of work and payment terms. MAGIC Grants will provide this contract.
+
+As required by law, contractors and employees will be required to provide certain documents to MAGIC Grants. This information will not be shared publicly or with the MAGIC Fund Committee to the extent allowable by law. MAGIC Grants will determine what documents need to be provided. MAGIC Grants may be legally required to publicly disclose compensation that meets certain criteria; please ask the board for specifics if this applies.
+
+## Grant Applications and MAGIC Fund Use
+
+The MAGIC Fund Committee shall decide how to use the MAGIC Fund, including the application process and deadlines (if any) for grant applications. MAGIC Grants encourages consistency for most grants while still allowing for agile grants as appropriate.
+
+The MAGIC Fund Committee requires at least **3 votes** to approve grant disbursements and investment decisions. A committee quorum requires at least **3 committee members** to be present. These can be changed by the board (for example if a larger committee is desired). The board may optionally use a binding or a non-binding election to collect feedback on potentially changing these thresholds.
+
+Selected MAGIC Fund investment decisions must comply with the MAGIC Investment Policy. MAGIC Grants allows MAGIC Funds to have wide discretion over their own investment policies so long as they are compatible with MAGIC’s Investment Policy.
```

Comment:
Are fund disbursements and grant disbursements separate? I am wondering how many votes would be needed to approve expenses for e.g. servers/CDN/etc.

(_Currently_ it is 1 lol)

Reply:
Grant disbursements are a subset of fund disbursements. This was originally written for a committee that would have more "bulk grants" than recurring expenses, and I can improve it if you like for your situation (or it can be improved later). The committee can vote to delegate responsibility to a committee chair for basic recurring expenses like servers/CDN.

In any case, the use of funds needs to be justified by the organization's mission, and there are requirements before money can be sent to someone else. For vendors like servers/CDN, you just need to explain why this vendor is being selected with some justification ("this is the most privacy-friendly" is an example) and then that will suffice.

Reply:
Okay so that sounds more a less what we were doing with OC then.